bin/upgrading/deadphrases.dat

```diff
@@ -1596,3 +1596,5 @@ general widget.search.msn
 general /manage/profile/index.bml.chat.msnusername
 general /profile.bml.im.msn
 general /profile.bml.label.msnusername
+
+general /site/opensource.bml.text.development.directions3
```

Comment:
Almost there! This string reference needs to be `/site/opensource.bml.development.directions3`, not `/site/opensource.bml.text.development.directions3`.  If you'll remove the .text portion I can get this in!